be/src/runtime/data_stream_sender.h

```diff
@@ -204,7 +199,11 @@ class DataStreamSender : public DataSink {
 
         // TODO(zc): initused for brpc
         PUniqueId _finst_id;
-        PRowBatch _pb_batch;
+
+        PRowBatch* _ch_cur_pb_batch;
+        PRowBatch _ch_pb_batch1;
```

Comment:
Add some comment to explain why named batch1 and batch2, or rename to a meaningful name?

Reply:
Comment added